src/components/SearchComponent.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import {View, TouchableOpacity, StyleSheet} from 'react-native'
+import {TextInput, IconButton, Surface} from '@react-native-material/core'
+import Icon from 'react-native-vector-icons/MaterialIcons'
+
+const SearchComponent = () => {
+  return (
+    <View style={{flexDirection: 'row'}}>
+      <TextInput
+        variant="outlined"
+        label="Search"
+        style={{margin: 10, width: '75%'}}
```

Comment:
a good practice is set 100% in width property and the reduce the space in the parent component (view container)

Reply:
thanks sir